spec/models/emotion_spec.rb

```diff
@@ -78,7 +86,11 @@
       let!(:existing_emotion) { create(:emotion, word: 'happy', category: 'positive', public: false) }
       let(:emotion_params) { { emotion: { word: 'happy', category: 'positive', public: false } } }
 
-      it 'does not create a new Emotion' do
+      it 'creates with existing emotion word and new category' do
+        expect { Emotion.create(word: 'happy', category: 'negative', public: false) }.to change(Emotion, :count)
+      end
+
+      it 'creates with existing emotion word and category' do
```

Comment:
the description of the test not matching with test content. It should be `it 'does not create with existing emotion word and category'`